data/tokens.json

```diff
@@ -1499,30 +1499,6 @@
       }
     }
   },
-  "Accordion": {
-    "gtmds_helper_accordion_border_open": {
-      "value": {
-        "color": "{gtmds_color_gray-4}",
-        "width": "{gtmds_border-width-base}"
-      },
-      "type": "border"
-    },
-    "gtmds_helper_accordion_heading_border_open": {
-      "value": {
-        "color": "{gtmds_color_gray-4}",
-        "width": "{gtmds_border-width-base}"
-      },
-      "type": "border"
-    },
-    "gtmds_helper_accordion_color_background_hover": {
-      "value": "{gtmds_color_blue-1}",
-      "type": "color"
```

Comment:
does the accordion has a background, or is it transparent? If there is background-color , we should define it as a token as well.

Reply:
We should also add a focus state. Is there also a disabled and error?